### ISSUE TITLE
Please disable all personal actionlists of the author

### ISSUE THREAD

**Issue by coezbek** [sl5net/global-IntelliSense-everywhere#8]
It seem that a lot of very specific ActionLists are currently shipped with GI which might not be applicable for every user. For example there is some banking stuff, Web.de, StackOverflow, etc.

These lists should be disabled by default or just shipped as examples.

**Comment by sl5net** [comment]
Yes currently GI shipp it as examples. All ActionLists are just examples. Yes might not be applicable for every user.
Theoretically, you can completely delete the https://github.com/sl5net/global-IntelliSense-everywhere/tree/master/ActionLists folder. Then no list will work. Everything in this order is, more or less, an example.
Thanks for your tip, I will continue to thank you. Maybe it should be better described.
